### ISSUE TITLE
Accessing search results not working well

### ISSUE THREAD

**Issue by jcowey** [papyri/navigator#46]
Searches involving `THEN` show a hit. But if you click on the hit it returns an `HTTP Status 400 – Bad Request`

So
![Bildschirmfoto 2019-05-17 um 14 21 55](https://user-images.githubusercontent.com/612992/57927782-6031d800-78af-11e9-98e0-36c9d364dc29.png)

Then 
![Bildschirmfoto 2019-05-17 um 14 23 54](https://user-images.githubusercontent.com/612992/57927810-75a70200-78af-11e9-9796-3796b2e17983.png)
Which uses the URL:

`http://papyri.info/search?STRING=(υμων+THEN+δεσποτων)~1chars&no_caps=on&no_marks=on&target=text&DATE_MODE=LOOSE&DOCS_PER_PAGE=15`

returns
![Bildschirmfoto 2019-05-17 um 14 25 15](https://user-images.githubusercontent.com/612992/57927883-a4bd7380-78af-11e9-8532-10b926cb1f47.png)
where the URL is

`http://papyri.info/ddbdp/p.grenf;1;60?rows=2&start=0&fl=id,title&fq=untokenized_ia:/@υμων.{1,1}δεσποτων@/&p=0&t=0`

**Comment by jcowey** [comment]
If I do the same using litpap
the URL is
`http://litpap.info/ddbdp/p.grenf;1;60/?q=REGEX%3A%28%E1%BD%91%CE%BC%E1%BF%B6%CE%BD%5Cs%2B%CE%B4%CE%B5%CF%83%CF%80%CE%BF%CF%84%E1%BF%B6%CE%BD%29&rows=2&start=0&fl=id%2Ctitle&fq=%7B%21regexp+cache%3Dfalse+qf%3D%22untokenized_ia%22%7D%5E.*%CF%85%CE%BC%CF%89%CE%BD.%7B1%2C1%7D%CE%B4%CE%B5%CF%83%CF%80%CE%BF%CF%84%CF%89%CE%BD.*%24&sort=series+asc%2Cvolume+asc%2Citem+asc&p=0&t=0`

and the text appears

![Bildschirmfoto 2019-05-17 um 14 29 07](https://user-images.githubusercontent.com/612992/57928125-31683180-78b0-11e9-903f-51ebf025154c.png)

Would you please take a look and try to fix this.

**Comment by jcowey** [comment]
This mail was sent to Rodney Ast.
"Since yesterday, we (Sophie Kovarik and me) have a problem with the Navigator. I checked it also with Laura Löser – just in case it was something in Vienna – and she has the same problem. When we use the search engine, it works perfectly. When we want to limit the number of results, for example by using the “Date on or after”, we get the wished / expected results. If we click on any of the results, however, it gives an error (see below), the same happens, when we use searches with AND or THEN. NOT, however, works perfectly."

I think this is an aspect of what I described above. I hope that once the bug is found all these problems with filters etc. will be dealt with in one pass.
